extra/alacritty.info

```diff
@@ -107,6 +107,6 @@ alacritty+common|base fragment for alacritty,
     Ms=\E]52;%p1%s;%p2%s\007, Se=\E[0 q, Ss=\E[%p1%d q,
     hs, dsl=\E]2;\007, fsl=^G, tsl=\E]2;,
     Smulx=\E[4\:%p1%dm,
-    Sync=\EP=%p1%ds\E\\,
+    Sync=\E[?2026%?%p1%{1}%-%tl%eh%;,
```

Comment:
Did you copy this from a different terminfo or write yourself?

Reply:
The entry is copied from [another terminal emulator](https://mitchellh.com/ghostty) which I’m helping to beta test. I didn’t write the original entry, but have since updated it.

Reply:
@chrisduerr foot's entry is the same here if it matters.

Reply:
I'm mostly just curious because terminfo syntax is a massive pita. But it looks about right just the trailing semicolon confused me.

> foot's entry is the same here if it matters.

I just checked and it's not. It's missing the trailing semicolon.

Reply:
Can you just remove the trailing `%;`? It's not that this is wrong a semicolon here should work just fine, but I don't think there's any benefit to having it so might as well reduce the statement's complexity.

Reply:
Wezterm has the same entry as foot, without the trailing semicolon. According to the [terminfo man page](https://man7.org/linux/man-pages/man5/terminfo.5.html ) the trailing `%;` is required for a conditional expression. Whether or not it actually matters in practice, though, I’m less sure.